Makefile

```diff
@@ -22,6 +22,11 @@ deps:
 	npm install
 	node_modules/gulp/bin/gulp.js
 
+deps3:
+	pip3 install -r requirements.txt
```

Comment:
I wanted to try and keep the environments separated since its only really in development where we'd use both python 2.X and 3.X at the same time